.github/workflows/crdb-api-client-npm-publish.yml

```diff
@@ -0,0 +1,110 @@
+# This workflow establishes two-step process of building and publishing `crdb-api-client` NPM package
+# when the produced outputs are changed comparing to last published version.
+# 1st step (first CI run) checks that built files contain changes comparing to previous version and
+# if NPM package version incremented:
+# - if yes, it publishes to NPM registry
+# - if no, new PR created (with incremented version in package.json file) which should be merged and
+# then second step is executed (runs workflow again) and publishes package.
+name: Publish crdb-api-client package to NPM registry
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish_crdb_api_client:
+    if: github.repository == 'cockroachdb/cockroach'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
A nit. Can you use the `actions/checkout@v4` - it uses the supported node version and more efficient IIRC.

Reply:
Done.